spec.html

```diff
@@ -30183,6 +30209,7 @@ <h1>
                 1. Set _k_ to _k_ + 1.
             1. Let _bodyParseString_ be the string-concatenation of 0x000A (LINE FEED), _bodyString_, and 0x000A (LINE FEED).
             1. Let _sourceString_ be the string-concatenation of _prefix_, *" anonymous("*, _P_, 0x000A (LINE FEED), *") {"*, _bodyParseString_, and *"}"*.
+            1. Perform ? HostEnsureCanCompileStrings(_currentRealm_, _parameterStrings_, _bodyString_, _sourceString_, ~function~, _parameterArgs_, _bodyArg_).
```

Comment:
Can you elaborate more on why do we need to pass two strings for the body? _bodyString_ and _sourceString_ are different, but I'm not sure why would the Host need _bodyString_ that _sourceString_  and _bodyArg_ cannot provide?

Reply:
Delegates decided against allowing the fully constructed codeString so this is no longer relevant.